config/document_types.yml

```diff
@@ -27,12 +26,12 @@
       type: multi_tag
       document_type: world_location
 
-- id: press_release
-  label: Press release
-  path_prefix: /government/news
-  images: true
+publication: &publication
+  pre_release: true
```

Comment:
It seems a shame that we have to manage this pre_release flag in two places (here and document_type_selection).

Could we have it so a document type selection is able to look up the corresponding document_type and question whether that is pre release rather than needing to maintain two lists?

Reply:
Yeah. That is something that occurred to me too, after the functionality had to be duplicated  for DocumentTypes.

I'll take a look and see what I come up with.